doc/api/index.md

```diff
@@ -2251,8 +2691,13 @@ mode (see [loadVideo options](./loadVideo_options.md#prop-transport)).
 <a name="meth-getWantedBufferAhead"></a>
 ### getWantedBufferAhead #######################################################
 
-_return value_: ``Number``
-_defaults_: ``30``
```

Comment:
Don't you want to keep the default values ?

Reply:
Yes, but isn't it a loss for the doc readers? Typically, I suppose we probably had users that found the default value the wanted buffer ahead here, and not elsewere.

Reply:
I still don't know if the getter is the right place to add it. The setter makes as much sense to me.

Perhaps putting it in the API description of both would be better there?

Reply:
Done. I added it to the body of the documentation for both `setWantedBufferAhead` and `getWantedBufferAhead`